config/storybook-addon-knobs.d.ts

```diff
@@ -0,0 +1,29 @@
+declare module 'storybook-addon-knobs' {
+  import WrapStory from "../src/components/WrapStory.js";
```

Comment:
How does this work @Ephys? `WrapStory` has no declaration file, so wouldn't this throw an error?